spec/features/catalog_page_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Catalog Page', type: :feature do
```

Comment:
Wondering what you think about naming the test and file something more specific, like `facet_spec.rb` or `search_filters_spec.rb`?

Reply:
I had it facet specifically originally and went back and forth. I'll change it back. I do prefer smaller test files.

Reply:
Okay I landed on `facets_spec.rb`, let me know what you think.